content/docs/1.0.x/install/authenticate-cli-bridge/index.md

```diff
@@ -7,113 +7,123 @@ weight: 60
-$Env:KEPTN_ENDPOINT = 'http://<ENDPOINT_OF_API_GATEWAY>/api'
-```
+   **Note**: If you receive a warning
+   `Using a file-based storage for the key because the password-store seems to be not set up.`,
+   it means that no password store could be found in your environment.
+   In this case, the credentials are stored in `~/.keptn/.password-store` in your home directory.
 
-* Copy the following snippet and paste it in the PowerShell. The snippet retrieves the API token and sets the environment variable `KEPTN_API_TOKEN`:
+**Using Windows PowerShell**
 
-```
-$tokenEncoded = $(kubectl get secret keptn-api-token -n keptn -ojsonpath='{.data.keptn-api-token}')
-$Env:KEPTN_API_TOKEN = [System.Text.Encoding]::UTF8.GetString([System.Convert]::FromBase64String($tokenEncoded))
-```
+For the Windows PowerShell, a small script is provided
+that installs the `PSYaml` module and sets the environment variables.
 
-* To authenticate the CLI against the Keptn cluster, use the [keptn auth](../../reference/cli/commands/keptn_auth) command:
+1. Set the environment variable `KEPTN_ENDPOINT`:
 
-```
-keptn auth --endpoint=$Env:KEPTN_ENDPOINT --api-token=$Env:KEPTN_API_TOKEN
-```
+   ```
+   $Env:KEPTN_ENDPOINT = 'http://<ENDPOINT_OF_API_GATEWAY>/api'
+   ```
 
-</p>
-</details>
+1. Copy the following snippet and paste it in the PowerShell.
+   The snippet retrieves the API token and sets the environment variable `KEPTN_API_TOKEN`:
 
-<details><summary>Command Line</summary>
-<p>
+   ```
+   $tokenEncoded = $(kubectl get secret keptn-api-token -n keptn -ojsonpath='{.data.keptn-api-token}')
+   $Env:KEPTN_API_TOKEN = [System.Text.Encoding]::UTF8.GetString([System.Convert]::FromBase64String($tokenEncoded))
+   ```
 
-In the Windows Command Line, a couple of steps are necessary.
+1. To authenticate the CLI against the Keptn cluster,
+   use the [keptn auth](../../reference/cli/commands/keptn_auth) command:
 
-* Set the environment variable `KEPTN_ENDPOINT`:
+   ```
+   keptn auth --endpoint=$Env:KEPTN_ENDPOINT --api-token=$Env:KEPTN_API_TOKEN
+   ```
 
-```console
-set KEPTN_ENDPOINT=http://<ENDPOINT_OF_API_GATEWAY>/api
-```
+**Using the Windows Command Line**
 
-* Get the Keptn API Token encoded in base64:
+In the Windows Command Line, do the following:
 
-```console
-kubectl get secret keptn-api-token -n keptn -ojsonpath={.data.keptn-api-token}
-```
+1. Set the environment variable `KEPTN_ENDPOINT`:
 
-```console
-abcdefghijkladfaea
-```
+   ```
+   set KEPTN_ENDPOINT=http://<ENDPOINT_OF_API_GATEWAY>/api
+   ```
 
-* Take the encoded API token - it is the value from the key `keptn-api-token` (in this example, it is `abcdefghijkladfaea`) and save it in a text file, e.g., `keptn-api-token-base64.txt`
+1. Get the Keptn API Token encoded in base64:
 
-* Decode the file:
+   ```
+   kubectl get secret keptn-api-token -n keptn -ojsonpath={.data.keptn-api-token}
+   ```
 
-```
-certutil -decode keptn-api-token-base64.txt keptn-api-token.txt
-```
+   ```
+   abcdefghijkladfaea
```

Comment:
this is probably not right :)

Reply:
Sorry, that is what is in both old versions.  What would be a better string?

Reply:
you can directly decode the token with `base64`:
```
kubectl get secret keptn-api-token -n keptn -ojsonpath={.data.keptn-api-token} | base64 -d
```